class-pull-hours-settings.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Class that defines plugin settings.
+ *
+ * @package MITlib Pull Hours
+ * @since 0.0.1
```

Comment:
I'm not sure if this matters, but this `@since` is difference here compared to in some of the other files in the same package here.

Reply:
Ultimately I'm not sure how much this parameter matters for our use cases. If you're stricken with curiosity to dig in further...

[WordPress' documentation standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/) adopt the conventions of [phpDocumentor](http://docs.phpdoc.org/). The `@since` tag is [specifically mentioned](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#since-section-changelogs), as a way of identifying when various features have been added to a project. [The full phpDoc specification for `@since` can be found here](http://docs.phpdoc.org/references/phpdoc/tags/since.html).

All of which is to say that varying values will be found around a given project as it grows over time. I'm probably going overboard here by varying values within a PR, though. 🤷‍♂️